hs_core/page_processors.py

```diff
@@ -120,6 +120,16 @@ def get_page_context(page, user, resource_edit=False, extended_metadata_layout=N
     topics = list(topics)  # force QuerySet evaluation
     content_model.update_relation_meta()
 
+    # identify whether Creators and Contributors are linked to active user accounts
+    creators = content_model.metadata.creators.all()
+    for creator in creators:
+        creator.is_active = True
+        id = re.sub("[^0-9]", "", creator.description)
+        if id.isnumeric():
+            user = user_from_id(id, raise404=False)
```

Comment:
@pkdash I feel like there has to be a better way than `user_from_id` to determine if an author/Creator is linked to an _active_ User? I didn't see a good way to do this...

Reply:
The description field on creator needs a hard look along with a rename.  It stores the creator's user id as a string like `/user/5`.  It should just store an integer and be named `hydroshare_user_id` or something similar.

I think this could be refactored into a function that takes a `hs_core.models.Party` type as a paramter and returns the user if active.

Reply:
@devincowan The build is failing due to a circular import in `hs_core/models.py`. Move the import` hs_core.hydroshare import user_from_id` to the` is_active()` method of the `Party` class.

Reply:
thank you @pkdash

Reply:
description has nothing to do with dublin core... it is just what was used in hydroshare.  hsmodels and hsclient will have to be updated.